src/main/scala/gestalt/Quote.scala

```diff
@@ -227,6 +231,65 @@ abstract class Quote(val t: Toolbox, val toolboxName: String) {
     case _ => t.Lit(name.value)
   }
 
+  /** lift modifiers */
+  def liftMods(mods: Seq[m.Tree]): t.Tree = {
+    mods match {
+      case Seq(quasi: Quasi) => return liftQuasi(quasi)
+      case _ =>
+        if (!isTerm) {
+          t.error("Match modifiers in syntax is problematic and not supported.", enclosingTree)
```

Comment:
This is brilliant! This forces to handle modifiers that would be easily forgotten, leading to all sorts of bugs.
Let's be nice and explain to the macro writer what to do now.
"Match modifiers in syntax is problematic and not supported. Match the modifiers with a variable instead."
Or even:
 ".Match modifiers in syntax is problematic and not supported. Match the modifiers with a variable instead or $_ to ignore them."